src/components/Form/FieldSelect/docs.stories.tsx

```diff
@@ -12,7 +12,7 @@ export default {
 type FormSchema = z.infer<ReturnType<typeof zFormSchema>>;
 const zFormSchema = () =>
   z.object({
-    color: z.enum(['red', 'green', 'blue']),
```

Comment:
We don't want it to be nullish for most of the stories. For the change you made, please create a new schema